packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -2114,6 +2114,273 @@ void main() {
     expect(find.textContaining('15').last, findsOneWidget);
     expect(find.text('2018'), findsOneWidget);
   });
+
+  testWidgets('CupertinoDatePicker selectionOverlayBuilder with monthYear mode',
+      (WidgetTester tester) async {
+
+    final Widget selectionOverlay = Container(
+      color: const Color(0x12345678),
+    );
+
+    // For mode = CupertinoDatePickerMode.monthYear
+    await tester.pumpWidget(
+      CupertinoApp(
+        home: Center(
+          child: CupertinoDatePicker(
+            mode: CupertinoDatePickerMode.monthYear,
+            onDateTimeChanged: (_) {},
```

Comment:
This underscore should be written out instead: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-anonymous-parameter-names